packages/datagateway-common/package.json

```diff
@@ -42,14 +42,15 @@
     "@types/react": "^16.9.41",
     "@types/react-router-dom": "^5.1.5",
     "@types/react-virtualized": "^9.21.10",
-    "@typescript-eslint/eslint-plugin": "^3.8.0",
-    "@typescript-eslint/parser": "^3.6.0",
+    "@typescript-eslint/eslint-plugin": "^4.0.0",
+    "@typescript-eslint/parser": "^4.0.0",
     "babel-eslint": "10.1.0",
     "enzyme": "^3.10.0",
     "enzyme-adapter-react-16": "^1.14.0",
     "enzyme-to-json": "^3.3.5",
+    "eslint": "^7.5.0",
     "eslint-config-prettier": "^6.11.0",
-    "eslint-config-react-app": "^5.2.1",
+    "eslint-config-react-app": "^6.0.0",
     "eslint-plugin-cypress": "^2.11.1",
     "eslint-plugin-flowtype": "^5.1.3",
```

Comment:
`eslint-plugin-flowtype` needs to be updated to`^5.2.0` at least, and `eslint-plugin-react` to `^7.20.3` at least to fix two more warnings. Don't know why this only warns in common but should probably also be applied in the other repos as well.

Reply:
Ah yeah I see we're getting incorrect dependency warnings, I'll get those too. I think it only warns you once for each dependency irrespective of the number of places it's got the wrong version (i.e. when you change a version in common, it won't warn you there but it will warn you that it's a problem in dataview, then when you fix it there it warns you somewhere else etc.)